indico/web/client/js/utils/redux.js

```diff
@@ -108,12 +108,14 @@ export function submitFormAction(submitFunc, requestAction, successAction, error
 
 export function ajaxAction(requestFunc, requestAction, successAction, errorAction, transformData = d => d) {
     return async (dispatch) => {
-        dispatcher(dispatch, requestAction);
+        if (requestAction) {
+            dispatcher(dispatch, requestAction);
+        }
         let response;
         try {
             response = await requestFunc();
         } catch (error) {
-            if (_.get(error, 'response.status') === 418) {
+            if (errorAction && _.get(error, 'response.status') === 418) {
```

Comment:
That looks wrong. If you get an error back you still go into the else block where you try to dispatch the action that is not set. I would only make the dispatch conditional but keep the return anyway.